docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+name: zarrcade
```

Comment:
@krokicki, Is this is a leftover from copy pasting?

Reply:
Yes, thanks.